### PR TITLE
added pgpool project

### PR DESCRIPTION
**Pull request by Paulius0112**

### CODE REVIEW

**Comment by Paulius0112** [comment]
fixing errors

File: pgpool/roles/pgpool/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+pool_passwd: root
+connection_cache: "off"
+replication_mode: "off"
+replicate_select: "off"
+load_balance_mode: "on"
+master_slave_mode: "on"
+master_slave_sub_mode: 'stream'
+sr_check_user: 'pgpool'
+wd_lifecheck_user: 'pgpool'
+pgpool_db_hosts: ['35.234.144.249', '34.105.206.15']
```
Comment (Zygimantass) [inline pgpool/roles/pgpool/defaults/main.yml:11]: what are these hardcoded IPs? there shouldn't be any hardcoded / public IPs used.

File: pgpool/roles/postgres/templates/postgres_master.j2 [hunk 2]
```diff
@@ -0,0 +1,603 @@
+# -----------------------------
```
Comment (Zygimantass) [inline pgpool/roles/postgres/templates/postgres_master.j2:1]: can you delete the commented out entries?

File: pgpool/roles/deploy_agent/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+- name: pull an image
+  docker_image:
+    name: syntropynet/agent:stable
```
Comment (Zygimantass) [inline pgpool/roles/deploy_agent/tasks/main.yml:3]: change this to latest

File: pgpool/roles/deploy_agent/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  docker_container:
+    name: syntropy-agent
+    state: absent
+    image: syntropy-agent
+
+- name: Deploy Syntropy Agent
+  docker_container:
+   name: syntropy-agent
+   restart_policy: unless-stopped
+   image: syntropynet/agent:stable
```
Comment (Zygimantass) [inline pgpool/roles/deploy_agent/tasks/main.yml:15]: change here too

**Comment by Paulius0112** [comment]
Final changes to pgpool task

**Comment by Zygimantass** [comment]
looks good to me

**Pull request merged**